test/test_lyrics.py

```diff
@@ -524,7 +523,32 @@ def test_json(self, mock_fetch_url, mock_scrape):
     # TODO: add integration test hitting real api
 
 
-# test Tekstowo
+@unittest.skipUnless(
+    os.environ.get('INTEGRATION_TEST', '0') == '1',
+    'integration testing not enabled')
+class GeniusIntegrationTest(GeniusBaseTest):
+    def setUp(self):
+        """Set up configuration"""
+        GeniusBaseTest.setUp(self)
+        genius_access_token = os.getenv('GENIUS_ACCESS_TOKEN')
+        beets_config = IncludeLazyConfig('beets')
+        beets_config.add(
+            {
+                'genius_api_key': genius_access_token
+            }
+        )
```

Comment:
This is an interesting tactic to get a "temporary" configuration for a test! It seems to work just fine. But FYI, we also have a general mechanism for doing this that we could reuse:
https://github.com/beetbox/beets/blob/36ea6f0777cb36ad67c01ac06e2f738754de7f52/test/_common.py#L171-L177

I'm fine keeping this tactic if you like—either one works, really.